utils/scan_parameters.py

```diff
@@ -188,6 +212,20 @@ def RunProjs(projs,hlsdir,kerasdir):
                 subprocess.call("rm -rf %s" % autopilot,
                             stdout = outlog, stderr=errlog, shell=True)
 
+            subprocess.call('mkdir %s/keras-to-hls/results_%s'%(hlsdir,k),
+                            stdout = outlog, stderr=errlog, shell=True)      
+         
+            if os.path.exists(output_filename):
+             subprocess.call('cp %s %s/keras-to-hls/results_%s/.'%(output_filename,hlsdir,k),
+                            stdout = outlog, stderr=errlog, shell=True)       
+              
+            if os.path.exists(report_filename):
+             subprocess.call('cp %s %s/keras-to-hls/results_%s/.'%(report_filename,hlsdir,k),
```

Comment:
Here, we should keep the whole directory of report if we are going to delete the whole project folder. There are the XML for each steps, which will be useful for detail performance study. The RPT file are not used now, but does contain more information.

Reply:
Ok. For the moment I was using only those two files. If you tell me which other files you think we might need I'll add them.

Reply:
I would suggest to copy the report folder to the result_%s . There are no specific filename, as the filename depending on the NN layer, and mostly randomly named by HLS

Reply:
Ok. I'll have a look first at how much space the report folder is taking and see if we can optimize it.